packages/tangle/messagefactory.go

```diff
@@ -324,9 +324,11 @@ var ZeroWorker = WorkerFunc(func([]byte) (uint64, error) { return 0, nil })
 // ReferencesFunc is a function type that returns like references a given set of parents of a Message.
 type ReferencesFunc func(strongParents MessageIDs, issuingTime time.Time, tangle *Tangle) (references ParentMessageIDs, err error)
 
-// PrepareReferences is an implementation of LikeReferencesFunc.
+// PrepareReferences is an implementation of ReferencesFunc.
 func PrepareReferences(strongParents MessageIDs, issuingTime time.Time, tangle *Tangle) (references ParentMessageIDs, err error) {
 	references = NewParentMessageIDs()
+	// TODO: these messages should be removed from the tips and their parents re-added.
```

Comment:
Don't we want to implement this too before merging?

Reply:
Done